### PR TITLE
fix: Update light and dark tooltip colors

### LINKED ISSUE #897: Info tooltip: colors contrast
**Description**
As info tooltip colors are not contrasting enough, the design team slightly updated colors now.
image
image

**Acceptance Criteria**
- [ ] Icon and label colors in hover/not hover state in light/dark themes should match updated designs

### PR DESCRIPTION
**Pull request by solid-yevheniiashchypeleva**
Fixes #897

### CODE REVIEW

File: metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart [hunk 1]
```diff
@@ -420,7 +417,7 @@ class DarkMetricsThemeData extends MetricsThemeData {
               inactive: CirclePercentageStyle(
                 strokeColor: _inactiveBackgroundColor,
                 backgroundColor: _inactiveBackgroundColor,
-                valueColor: _focusedBorderColor,
+                valueColor: ColorConfig.shimmerColor,
```
Comment (solid-vladlaba) [inline metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart:420]: Seems like this color doesn't match the design 
image

**Review by solid-vladlaba — CHANGES_REQUESTED** [review]
Well done, @solid-daniyilyevtyushkin @solid-yevheniiashchypeleva! I've added one small suggestion below - please let me know what you think!

File: metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart (hunk 1 above)
Comment (solid-daniyilyevtyushkin) [reply metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart:420]: This color is defined in `icons/unknown_status.svg`, but we'll update it, though.

Снимок экрана 2020-12-16 в 09 50 30

**Review by solid-vladlaba — APPROVED** [review]
LGTM, thanks @solid-daniyilyevtyushkin @solid-yevheniiashchypeleva!:rocket:

**Pull request merged**